packages/@n8n/nodes-langchain/nodes/llms/LMChatOpenAi/LmChatOpenAi.node.ts

```diff
@@ -186,6 +189,203 @@ export class LmChatOpenAi implements INodeType {
 					},
 				},
 			},
+			{
+				displayName: 'Built-in Tools',
+				name: 'builtInTools',
+				placeholder: 'Add Built-in Tool',
+				type: 'collection',
+				default: {},
+				options: [
```

Comment:
Shouldn't `builtInTools` be inside options as most users probably won't need to touch it